fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/LogicalMetric.java

```diff
@@ -80,6 +80,28 @@ public LogicalMetric(TemplateDruidQuery templateDruidQuery, ResultSetMapper calc
         this(templateDruidQuery, calculation, name, name, DEFAULT_CATEGORY, name);
     }
 
+    /**
+     * Constructor. Builds a Logical Metric whose instance variables are provided by a LogicalMetricInfo object.
+     *
+     * @param templateDruidQuery  Query the metric needs
+     * @param calculation  Mapper for the metric
+     * @param logicalMetricInfo  Logical Metric info provider
+     */
+    public LogicalMetric(
```

Comment:
It might be nice to implement this constructor and have everything else call this one.

This would require changing this constructor `LogicalMetricInfo(String name, String longName)` to `LogicalMetricInfo(String name, String description)` but that might be what we want since that's what `LogicalMetric` allows

Reply:
Let's go ahead and do this to push people toward the LogicalMetricInfo constructor